lib/client/commands/login.go

```diff
@@ -116,5 +140,24 @@ func (l *Login) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+        // TODO(isaac): wrap below in a function and call via defer
```

Comment:
I never really looked at this myself, but it looks like defered funcs do not get called https://stackoverflow.com/questions/63257505/does-a-deferred-restore-run-if-you-ctrl-c-out-of-an-app

I'm curious the reason to run that defered though - is the idea to just run it async (seeing that we don't really care about any errors that may pop up)? If so, I'd recommend using a goroutine instead.

Reply:
It looks like the rest of the code just assumes the user isn't going to ctrl-c before the other tasks complete (write the file to `~/.config`, etc..), so probably overly cautious here.